roles/cache_clear/cache_clear-opcache/defaults/main.yml

```diff
@@ -4,7 +4,7 @@ cache_clear_opcache:
   # eg.
   # --fcgi=127.0.0.1:9000
   # Leave blank to use /etc/cachetool.yml
-  adapter: ""
+  adapter: "127.0.0.1:90{{ _php_version | replace('.','') }}"
```

Comment:
This isn't going to work, unfortunately. I would get rid of the "Leave blank" and comment it out, something like this:
```yaml
  # adapter: "127.0.0.1:9081" # Leave commented to automatically detect the adapter based on PHP version.
```